### DISCUSSION TITLE
Support for Ben Eater's 6502 computer

### DISCUSSION THREAD

**Discussion by patrickleboutillier** [cc65/cc65#1814] [Ideas]
Hi all,

I see references in a few places (Reddit, various forums) of people trying to use cc65 with Ben Eater's 6502 computer (https://eater.net/6502). 

If I attempt to add support to cc65 for that target, is there a chance that it would be accepted into a release?

Thanks,

Patrick

**Comment by mrdudz** [comment]
Am Donnerstag, 28. Juli 2022, 01:43:09 CEST schrieb Patrick LeBoutillier:
> Hi all,
>
> I see references in a few places (Reddit, various forums) of people trying
> to use cc65 with Ben Eater's 6502 computer (https://eater.net/6502).
>
> If I attempt to add support to cc65 for that target, is there a chance that
> it would be accepted into a release?

sure :)

--

http://hitmen.eu                 http://ar.pokefinder.org
http://vice-emu.sourceforge.net  http://magicdisk.untergrund.net

Foolish man give wife grand piano. Wise man give wife upright organ.
